charts/app-of-apps/values-staging.yaml

```diff
@@ -21,7 +21,7 @@ ckanHelmValues:
         title: "data.gov.uk"
       dbHost: "ckan-postgres.staging.govuk-internal.digital"
       redis:
-        host: "shared-redis-govuk.eks.staging.govuk-internal.digital"
+        host: "dgu-shared-redis"
```

Comment:
Just out of curiosity, is there a reason it has "shared" in the name?

Reply:
yes, it's been recently named as dgu-shared